### ISSUE TITLE
The performance on DB15K is not as good as reported in the paper

### ISSUE THREAD

**Issue by Anionionionio** [zjukg/K-ON#1]
The performance on DB15K is not as good as reported in the paper. Hello, great work. I used 8 A100 cards to run the dataset DB15K, and loaded the large model with torch_dtype=bf16. After 5 epochs, the best MRR result I got was only 0.29. Could you please tell me why this might be the case? LLM must be loaded with fp32?

**Comment by Anionionionio** [comment]
I find with torch_dtype=fp32, the mrr is also very low, about 0.30

**Comment by Anionionionio** [comment]
i find that model with single a100 gpu and smaller batch size is relatively higher? but still with mrr 0.35 not good as 0.38

**Comment by Cameliasss** [comment]
metoo! I strictly followed the setup from the paper to train on three datasets, but the results did not meet expectations. For example, on DB15K, I used 8 GPUs (H200), epoch=5, lr= 1e-4, per-device train batch size=12, gradient accumulation steps=8. The results were:
* MRR: 0.3552（0.3810）
* Hits@1: 0.2614（0.3013）
* Hits@3: 0.4018（0.4277）
* Hits@10: 0.5367（0.5359）
